### PR TITLE
Add alarm() and setitimer()

### PR DESCRIPTION
**Pull request by AlpacaMax**

### CODE REVIEW

File: posix/alarm.c [hunk 1]
```diff
@@ -35,5 +35,5 @@ alarm (seconds)
 }
 libc_hidden_def (alarm)
 
-stub_warning (alarm)
+// stub_warning (alarm)
```
Comment (rennergade) [inline posix/alarm.c:38]: we probably dont need to still comment this

File: sysdeps/nacl/alarm.c [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+unsigned int alarm(unsigned int seconds) {
+  struct itimerval timer;
+  struct itimerval old_timer;
+
+  timer.it_value.tv_sec = seconds;
+  timer.it_value.tv_usec = 0;
+  timer.it_interval.tv_sec = 0;
+  timer.it_interval.tv_usec = 0;
+
+  setitimer(ITIMER_REAL, &timer, &old_timer);
```
Comment (rennergade) [inline sysdeps/nacl/alarm.c:13]: did you confirm this routes to your settimer function?

Comment (rennergade) [reply sysdeps/nacl/alarm.c:13]: shouldn't it be the lind one?

**Review by rennergade — COMMENTED** [review]
Looks good. Confused by alarm.c and where it routes.

File: sysdeps/nacl/alarm.c (hunk 2 above)
Comment (AlpacaMax) [reply sysdeps/nacl/alarm.c:13]: Adding some print statements shows that the setitimer I implemented is called when calling alarm.

**Pull request closed without merging**